scene/3d/occluder_instance_3d.cpp

```diff
@@ -719,6 +724,19 @@ Ref<Resource> OccluderInstance3D::_get_editable_3d_polygon_resource() const {
 	return occluder;
 }
 
+void OccluderInstance3D::_notification(int p_what) {
+#ifdef TOOLS_ENABLED
```

Comment:
This `#ifdef` is required because https://github.com/godotengine/godot/issues/62020 isn't implemented yet.